packages/common/src/webrtc/Peer.ts

```diff
@@ -15,6 +15,14 @@ export default class Peer {
   private _constraints: { offerToReceiveAudio: boolean, offerToReceiveVideo: boolean }
   private _negotiating: boolean = false
 
+  resetNegotiating() {
+    this._negotiating = false
+  }
+
+  isNegotiating() {
+    return this._negotiating
+  }
```

Comment:
Should we convert this into a getter?
```suggestion
  get isNegotiating() {
    return this._negotiating
  }
```